package.json

```diff
@@ -92,6 +92,7 @@
     "@hawkrives/react-native-alphabetlistview": "1.0.0",
     "@hawkrives/react-native-alternate-icons": "0.4.7",
     "@hawkrives/react-native-sortable-list": "1.0.1",
+    "bluebird": "3.5.1",
```

Comment:
Can you use https://www.npmjs.com/package/p-props instead, please? It's a more focused dependency.